### ISSUE TITLE
Copy value in Debugger truncates value with ellipsis with Node

### ISSUE THREAD

**Issue by jeffmcaffer** [microsoft/vscode#122880]
- VS Code Version: 1.55.2
- OS Version: windows 19041

Steps to Reproduce:

1. Write some code that has a long string value in an variable
2. Debug said code with a breakpoint when the variable has the value
3. In the debugger's Variable pane, use the Copy value context menu option to copy the value of the string variable
4. Paste that copy into a normal text editor
5. Notice that the string value is truncated with ellipsis (...) in the middle of the string 

Desired outcome is that the full value of the string is copied into the copy buffer. 

As a poor substitute, increase the size of allowed full values to something large enough to be less frequent of an issue. I'm getting this truncation on a string of roughly 30 lines of generated code (of a normal-ish density).  Feels like a very small limit given available memory.

Does this issue occur when all extensions are disabled?: Yes/No
yes

**Comment by isidorn** [comment]
@jeffmcaffer this works nicely for me with js-debug. What debugger are you using, I believe the issue is that the debug extension you are using does not evaluate properly the element. Since we evaluate before we copy to make sure to get the full value

[Code pointer](https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/debug/browser/variablesView.ts#L441)

**Comment by jeffmcaffer** [comment]
hey @isidorn, How would I tell what version of the debugger I'm using? I just downloaded Code and ran. There did not appear to be a need to install a debugger for JS. 

I just tried to write a simple repo but it doesn't happen there. I does happen in the middle of my complex system but I can't share that. It's not critical but has been super useful to pause, copy the generated code and see if it's correct (e.g., compiles/runs). Unfortunately, with the `...` truncation, that fails. I'll see if there is something more specific going on.
